### ISSUE TITLE
[Feature Request] Setting to only enable segmented hotbar

### ISSUE THREAD

**Issue by csumpasd** [Lizard-Of-Oz/Inventorio#178]
Hi!

I have been really enjoying your mod, but since I have a 40% keyboard, I found the mod's segmented hotbar feature especially helpful. When playing against friends however (such as doig manhunts), where the toolbelt and such features would be overpowered, it'd be really great if there was a toggle to disable those(so basically everything else), while keeping the segmented hotbar on. I've tried to write a mod to do this myself but I'm not knowledgeable enough yet to do so.

It'd be awesome if this could be a feature in 2.0, but it's fine if not! 
Thanks for writing the mod and doing all this work :D

**Comment by Lizard-Of-Oz** [comment]
Modularity is the core thing of 2.0. You'll even be able to use this as a client-side only mod, but be warned: 2.0 is coming in a rather distant future.
Thanks for feedback.

**Comment by csumpasd** [comment]
Hi! 

Sorry for reopening this issue.
You, and this segmented hotbar idea motivated me to write my own mod. 
I used the fabric wiki and your source code to understand how minecraft modding, and especially, mixins work.
I used java, and didn't copy-paste from your code, but since I couldn't find an indication of any license in your project, I wanted to ask if you're okay with me essentially taking the basic idea from your mod. 

Currently I plan to build upon the idea, add features to it, and extend the key handling to the main inventory so it's consistent. I'm not planning on writing an inventory rework like yours.

I've added a text at the bottom of the mod description saying how it is heavily inspired by Inventorio.
Here's the repo: https://github.com/csumpasd/segmented

If you'd rather talk about this privately, send me an email at marton @ csutora . com
Either way, thanks for inspiring me to start modding, it's really fun so far :D
